api/proto/agent/artifact.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3"
+
+message Artifact {
+    bytes logs = 1; 
+    optional repeated bytes files = 2;
```

Comment:
Are you intending to send all generated files from the pipeline? Artifacts should stay in the CI and what you want to provide back is a URL to them.